app/views/admin/websites/_website_personas.html.erb

```diff
@@ -0,0 +1,42 @@
+
+  <div id="website-personas-div" class="grid-row" data-value="<%= admin_website_path(@website) %>">
+    <div class="grid-col-12">
+      <h3>Website Personas</h3>
+      <p class="website-persona-list">
+      <ul class="usa-list usa-list--unstyled">
+        <% @website.website_personas.each do |persona| %>
+        <li style="margin-bottom: 3px;">
+          <span class="usa-tag">
+             <%= persona.name %>
+          </span>
+          &nbsp;
+          <a href="javascript:void(0);" class="remove-persona-link" data-value="<%= persona.id %>">
+            <span class="fa fa-trash"></span>
+          </a>
+        </li>
+          <% end %>
+      </ul>
+      <p>
+    </div>
+  </div>
+  <%= select_tag :website_persona_id, options_for_select(@website_persona_options.map { |p| [p.name, p.id] }), prompt: "Choose a Persona", class: "usa-select add-website-persona" %>
+<script>
```

Comment:
wrap the query in a `$(function() {
  // the jquery code goes here. otherwise, it may not load (due to browser race conditions)
})`